Make.config

```diff
@@ -93,6 +93,13 @@ OSX_SDK_VERSION=10.15
 WATCH_SDK_VERSION=6.0
 TVOS_SDK_VERSION=13.0
 
+# Maximum OS versions for running XI/XM apps.
```

Comment:
We already have this logic in master, but slightly different: https://github.com/xamarin/xamarin-macios/blob/2adecee3d98ad93599141db53376971d4eb73633/Make.config#L92-L97

Could you make it identical (or as close as possible)? That will hopefully make merging this branch into master later easier.

Also you're assigning strings to the MACOS/WATCHOS/TVOS variables, not version represented by those strings.

Reply:
🤦‍♂ I forgot I added that to master